docs/proposals/last-operation-description.md

```diff
@@ -0,0 +1,68 @@
+# Last Operation Description
+
+## Introduction
+As per the OpenService Broker API spec, the last operation response can contain
+a description as well as a status [last operation response](https://github.com/openservicebrokerapi/servicebroker/blob/v2.12/spec.md#body).
+This can provide useful information about what is happening during a broker action along with details of the overall progress.
+UI implementors could make valuable use of this information
+to provide a progress bar or just simply a log of actions to show progress.
+```
+60%: succesfully created realm in keycloak
+```
+
+## Problem Description
+Currently we are only making use of the status field: 
+[last operation in broker.go](https://github.com/openshift/ansible-service-broker/blob/master/pkg/broker/broker.go#L1311)
+The difficulty is around how we get that detail back from the pod.
+Without the description, we limit the feedback that can be provided to the user.
+
+
+## Proposed Implementation
+Create an append only log file in a specified location ```/var/log/last_operation```, which is collected periodically and also a final time after
```

Comment:
terms: 
final operation - the operation that indicates the apb is 100% complete
last_operation - the most recent operation an apb performed

One thing I like about waiting to collect the final operation is that we know the apb is complete.  That might come in handy with dependencies.  On the other hand, does the broker really need the final operation if it's only a description?  How do you view the importance of the final operation?